README.md

```diff
@@ -22,9 +22,28 @@ model_path = cached_path("https://github.com/PathwayCommons/pathway-abstract-cla
+    "Regulatory T-cells (Tregs) are central in the maintenance of homeostasis and resolution of inflammation. However, the mechanisms that govern their differentiation and function are not completely understood. Herein, we demonstrate a central role for the lipid mediator biosynthetic enzyme 15-lipoxygenase (ALOX15) in regulating key aspects of Treg biology. Pharmacological inhibition or genetic deletion of ALOX15 in Tregs decreased FOXP3 expression, altered Treg transcriptional profile and shifted their metabolism. This was linked with an impaired ability of Alox15-deficient cells to exert their pro-resolving actions, including a decrease in their ability to upregulate macrophage efferocytosis and a downregulation of interferon gamma expression in Th1 cells. Incubation of Tregs with the ALOX15-derived specilized pro-resolving mediators (SPM)s Resolvin (Rv)D3 and RvD5n-3 DPA rescued FOXP3 expression in cells where ALOX15 activity was inhibited. In vivo, deletion of Alox15 led to increased vascular lipid load and expansion of Th1 cells in mice fed western diet, a phenomenon that was reversed when Alox15-deficient mice were reconstituted with wild type Tregs. Taken together these findings demonstrate a central role of pro-resolving lipid mediators in governing the differentiation of naive T-cells to Tregs."
+]
+
+texts = [title + " [SEP] " + abstract for title, abstract in zip(titles, abstracts)]
+predictions = model.predict(texts))
+assert predictions == [1,0]
+print("Articles correctly classified")
```

Comment:
This is going to print regardless of the truth value of the assert. I would remove
```suggestion
```

Reply:
Yeah I've modified my initial test file (classify_article.py) to run the code from this quickstart to ensure it works. For predictions ==[1,0] (expected) , line 46 gets printed. If I change to predictions == [0,1] I get an AssertionError and 46 does not print.

Reply:
Ah, I see. I think the `print` statement is superfluous but I won't fight you if you want to keep it. If you want to address the other outstanding comments then this can be merged.

Reply:
I mean, to be fair, I do see your argument there. I think I'll make that change, as well as the one we were discussing earlier. At the end of the day, this is an internal tool, so we have lots of opportunity to clarify and explain things as well as collect and address feedback. So you're probably right that a hand-holding but less minimalistic ReadMe isn't optimal. Thanks for the feedback 😃 .